src/text.zig

```diff
@@ -5,12 +5,14 @@ const std = @import("std");
 const mem = std.mem;
 const testing = std.testing;
 
-pub fn label(ctx: *nk.Context, text: []const u8, flags: nk.Flags) void {
-    return c.nk_label(ctx, nk.slice(text), flags);
+const Type = c.nk_text_alignment;
```

Comment:
Looking at Nuklear, the `nk_text_alignment` seems to be a bitfield, so we probably wonna do something simular to the `PanelFlags` struct